docs/getting-started/unreal-editor/index.md

```diff
@@ -29,11 +29,11 @@ You can set this easily by putting the launch option in the steam launch options
 ![Steam Launch Options](assets/steam-launchoptions.png)
 
 ### Cooking
-PAYDAY 3 requires all of its files to be cooked,
+PAYDAY 3 requires all of its files to be cooked before they can be packaged into a mod,
 so you must cook your game files everytime you want to package and test your mod.
 
 You can cook your game files by opening the `File` menu and clicking `Cook Content for Windows`.
-Once cooked, your files will be located at `{UE4ProjectDir}/Saved/Cooked/WindowsNoEditor/`.
+Once cooked, your files will be located at `{UE4ProjectDir}/Saved/Cooked/WindowsNoEditor/PAYDAY3/Content`.
```

Comment:
I feel like the original path here is more accurate. If I recall correctly, that's the root of what your .pak should be right?

Reply:
Nope, the Path I provided (checked it while I wa smaking my own mod) is correct

Reply:
Ill leave this as is, checked it again and it is infact `{UE4ProjectDir}/Saved/Cooked/WindowsNoEditor/PAYDAY3/Content`